railties/test/application/rake/multi_dbs_test.rb

```diff
@@ -127,36 +127,36 @@ class AnimalsBase < ActiveRecord::Base
 
       test "db:create and db:drop works on all databases for env" do
         require "#{app_path}/config/environment"
-        ActiveRecord::Base.configurations[Rails.env].each do |namespace, config|
-          db_create_and_drop namespace, config["database"]
+        ActiveRecord::Base.configurations.configs_for(Rails.env).each do |db_config|
```

Comment:
I changed these because this is what is in the database tasks code so we ensure all this keeps working.